Dockerfile

```diff
@@ -25,7 +25,6 @@ RUN dnf install -y \
       findutils \
       openscap-scanner \
```

Comment:
This could be removed as well; we no longer support HasMinimalVulns check

Reply:
Fair! I was trying to be cautious here because I thought we might reintroduce it at some point. Either way, I've removed this dependency - we can re-add it later if we need it.